src/TLuaInterpreter.cpp

```diff
@@ -3680,6 +3680,107 @@ int TLuaInterpreter::setMapWindowTitle(lua_State* L)
     return 1;
 }
 
+// Documentation: https://wiki.mudlet.org/w/Manual:Lua_Functions#setupMapSymbolFont
+int TLuaInterpreter::setupMapSymbolFont(lua_State* L)
+{
+    Host& host = getHostFromLua(L);
+    if (!host.mpMap || !host.mpMap->mpRoomDB) {
+        lua_pushnil(L);
+        lua_pushstring(L, "no map present or loaded");
+        return 2;
+    }
+
+    QString fontName;
+    int s = 0;
+    if (!(lua_isnil(L, ++s) || lua_isstring(L, s))) {
+        lua_pushfstring(L, "setupMapSymbolFont: bad argument #%d type (symbol font name as string is optional {use nil for no change, empty string for default}, got %s!)", s, luaL_typename(L, s));
```

Comment:
```suggestion
        lua_pushfstring(L, "setupMapSymbolFont: bad argument #%d type (symbol font name as string is optional, got %s!)", s, luaL_typename(L, s));
```

Too much information, hard to understand it all when you're new to this, this way is better.

Reply:
But it is an important functional difference.

If you need to alter one of the *other* following parameters **but not change this setting** you need to use a `nil` here as a placeholder. Whereas using the empty string `""` is an explicit expression that the value is to be set to something - which has the meaning of whatever the default is (`Deja Vu Mono` IIRC). Mixing them up could produce unwanted results (changing back to the default instead of leaving the current setting alone) so it is not wise to leave the information out in the case of the user using a wrong thing altogether...

Reply:
Again, this fine difference will be lost on people. This is better to be communicated by examples in the manual that people can copy/paste.